firmware/application/tone_key.hpp

```diff
@@ -30,6 +30,9 @@ using namespace ui;
 
 namespace tonekey {
 
+#define TONE_FREQ_TOLERANCE_HZ 4.0
+#define TONE_DISPLAY_TOGGLE_COUNTER 2
+
 typedef int32_t tone_index;
```

Comment:
nit: not your change, but should `index` be signed?

Reply:
Currently a value of -1 is being used to indicate that the tone frequency wasn't found in the table.